app/src/main/java/org/wikipedia/analytics/eventplatform/MachineGeneratedArticleDescriptionsAnalyticsHelper.kt

```diff
@@ -23,7 +23,7 @@ class MachineGeneratedArticleDescriptionsAnalyticsHelper {
     }
 
     fun articleDescriptionEditingEnd(context: Context) {
-        log(context, composeGroupString() + ".end")
+        log(context, composeGroupString() + ".timeSpentMs:${timer.elapsedMillis}.end")
```

Comment:
Can `timeSpentMs` come at the end of the string, so it's consistent with the other places it's sent?

Reply:
Done @dbrant